moveit_ros/moveit_servo/test/pose_tracking_test.cpp

```diff
@@ -63,24 +63,48 @@ class PoseTrackingFixture : public ::testing::Test
   void SetUp() override
   {
     // Wait for several key topics / parameters
-    ros::topic::waitForMessage<sensor_msgs::JointState>("/joint_states");
-    while (!nh_.hasParam("/robot_description") && ros::ok())
+    
+    //TODO: Need to figure out replacement. How to use WaitSet?
+    
+    //rclcpp::topic::waitForMessage<sensor_msgs::msg::JointState>("/joint_states");       <--- from ROS1    
+    
+    
+    // rclcpp::Node temp_node_=rclcpp::Node("temp");
+    // rclcpp::Subscription<sensor_msgs::msg::JointState>::SharedPtr joint_state_sub_ = temp_node_.create_subscription<sensor_msgs::msg::JointState>("joint_states", 1, RCLCPP__ANY_SUBSCRIPTION_CALLBACK_HPP_);
+    // rclcpp::WaitSet::add_subscription()
+    // rclcpp::WaitResult()
+
+  
+      bool have_message = false;
+      auto sub = node_.create_subscription<sensor_msgs::msg::JointState>(
+          "joint_states", 1,
+          std::function<void(const sensor_msgs::msg::JointState::ConstSharedPtr)>(
+              [have_message](const auto msg) mutable { have_message = true; }));
+      while (!have_message)
+        ;
+    
+
+    while (!node_.has_parameter("/robot_description") && rclcpp::ok())
```

Comment:
Parameter names don't use "/" anymore. parameter separators are "." in ROS 2
```suggestion
    while (!node_.has_parameter("robot_description") && rclcpp::ok())
```

Reply:
However, we can probably remove this while loop. It shouldn't be necessary at all.